content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -21,20 +21,57 @@ toc: true
 
 All Chainguard Containers contain verifiable signatures and build-time SBOMs (software bills of materials), features that enable users to confirm the origin of each image built and have a detailed list of everything that is packed within.
 
-The following commands requires [Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. It will pull detailed information about all signatures found for the provided image.
+This guide outlines how you can use Cosign to verify container image signatures and attestations, as well as download container attestations. 
+
+## Prerequisites
+
+The following examples require [Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. The examples in this guide will pull detailed information about all signatures found for the provided image.
 
 ## Registry and Tags for Chainguard Containers
 
 Attestations are provided per image build, so you'll need to specify the correct tag and registry when pulling attestations from an image with `cosign`.
```

Comment:
I think there's a sentence missing here, there's nothing leading into the bullets.

Also, I wonder if we should mention that it's best practice to use digests when doing verification.

Reply:
you're right, good catch!